integrationTest/integration_test.go

```diff
@@ -601,6 +610,15 @@ func makeTests(t *testing.T) []*TestCase {
 		)
 	}
 
+	// DS
+	if !providers.ProviderHasCabability(*providerToRun, providers.CanUseRoute53Alias) {
```

Comment:
This looks like it was started and not finished. I'd love to have some integration tests to implement it for more providers.